Sources/PaymentsCore/WebAuthenticationSession.swift

```diff
@@ -17,6 +17,8 @@ public class WebAuthenticationSession: NSObject {
         authenticationSession.prefersEphemeralWebBrowserSession = true
         authenticationSession.presentationContextProvider = context
 
-        authenticationSession.start()
+        DispatchQueue.main.async {
+            authenticationSession.start()
+        }
```

Comment:
This fixes the threading warning discussed [here](https://github.com/paypal/iOS-SDK/pull/103#issuecomment-1355025133)